eth/downloader/peer.go

```diff
@@ -31,6 +31,9 @@ import (
 	"gopkg.in/fatih/set.v0"
 )
 
+// The impact a single measurement has on a peer's final throughput value.
+const throughputImpact = 0.1
```

Comment:
Would put this and all other values which affect downloader / peer algorithms in the same file. It'll make it easier to see all configurable values in one view.